.gitignore

```diff
@@ -14,3 +14,7 @@ Cargo.lock
 # Code coverage
 *.profraw
 tarpaulin-report.html
+
+# It normally does not make sense to share proptest regression files across
+# machines because they are largely specific to a machine's topology.
+proptest-regressions/
```

Comment:
Unfortunately, proptest's regression testing feature is only useful locally to a single machine.